locales/oc/date.po

```diff
@@ -92,31 +92,31 @@ msgid "December"
 msgstr "Decembre"
 
 msgid "_Mon"
-msgstr ""
+msgstr "_Dl"
```

Comment:
Better but still not perfect: example from french → lun. for lundi

See also the en date.po → Mon for monday

Reply:
We never use days in short =/
I can put Dil, Dim, Dim, Dij, Div, Dis, Dim but it's not that pretty :crying_cat_face:

Reply:
> We never use days in short =/

I need to check where and how they are used in Dotclear…

Reply:
Well it's not that bad in 3 letters

Reply:
Tell me, as long as it's merged I'm happy :)

Reply:
> > We never use days in short =/
> 
> I need to check where and how they are used in Dotclear…

En fait c'est utilisé pour afficher une date en clair. Je suppose qu'avoir le nom du jour en entier ne sera pas gênant, plutôt qu'une abréviation.

Du coup le mieux est de mettre 3 ou 4 caractères ou plus, du moment que ça reste compréhensible.

PS : Ne pas mettre le "underscore" dans la traduction.

Reply:
Voilà, je pense que pour un premier rafraîchissement on est pas mal !